remo/events/api/views.py

```diff
@@ -18,3 +30,62 @@ def retrieve(self, request, pk):
         serializer = EventDetailedSerializer(event,
                                              context={'request': request})
         return Response(serializer.data)
+
+
+class EventsKPIFilter(django_filters.FilterSet):
+    class Meta:
+        model = Event
+        fields = ['categories__name', 'campaign__name', 'country']
+
+
+class EventsKPIView(APIView):
+
+    def get(self, request):
+        """Returns serialized data for Events KPI"""
+
+        f = EventsKPIFilter(request.GET, queryset=Event.objects.all())
+        weeks = int(request.QUERY_PARAMS.get('weeks', KPI_WEEKS))
+
+        # Total calculations
+        total = f.qs.count()
+
+        # Quarter calculations
+        current_q = int(math.ceil(now().month/3.0))
+        q_start = datetime(now().year, current_q, 1)
+        total_q = f.qs.filter(start__gte=q_start).count()
```

Comment:
I have applied ``range`` to other references that this comment applied. I also updated the initial queryset in ``f.qs`` filtering only current or past events. We can leave this without any change.